resources/lang/de/routes.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+// Password Reminder Language Lines
+return [
+    '' => [
+        'locale' => 'de',
+        'resources' => [
+            'index' => 'Index',
```

Comment:
@floler : First of all, thanks you a lot for sending this PR. I have few remarks regarding ```resources/lang/de/routes.php``` file: 

This file is used to localize URLs, and we hold those related translations here. For example, for resources, where it should be ```http://localhost/en/resourceHere/create``` it will become ```http://localhost/de/resourceHere/erstellen``` in German. You can see all these final translations if you run ```./artisan route:list``` command. All URLs starting with ```en/``` prefix will be translated (of course you also need to add ```de``` to the list of languages inside ```app/config/app.php``` file).

For that reason: Could you please make all translations inside this file lower-case? I mean, it should be ```dateien``` instead of ```Dateien``` etc. Thanks.